components/profile_picture.tsx

```diff
@@ -38,9 +38,11 @@ export default class ProfilePicture extends React.PureComponent<Props> {
     };
 
     public hideProfilePopover = () => {
-        if (this.overlay.current && typeof this.overlay.current.hide === 'function') {
-            this.overlay.current.hide();
-        }
+        this.overlay.hide!();
```

Comment:
This sometimes failed on me and failing on [test](https://github.com/mattermost/mattermost-webapp/pull/4152#discussion_r343967171).  Should this be?
```javascript
if (this.overlay) {
    this.overlay.hide!();
}
```

Reply:
Yes, I believe overlay can be null. I'll add the check